### DISCUSSION TITLE
Feature Request: Add Support for Remote Image URLs Without Downloading

### DISCUSSION THREAD

**Discussion by TomoyaKuroda** [payloadcms/payload#8069] [Feature Requests & Ideas]
**Description:**

PayloadCMS currently allows users to download an image from a URL and use it in the CMS. However, I'd like to request a feature where we can specify a remote image URL and use it directly without downloading the file to the local server.

This would be especially useful for scenarios where the CMS is deployed on platforms like Vercel, which do not support file uploads. By allowing remote image URLs, we could utilize external storage or image hosting services without the need to store images locally.

**Use Case:**

- Deploying PayloadCMS on platforms that don’t allow file uploads (e.g., Vercel).
- Leveraging external image hosting services (such as AWS S3, Cloudinary, etc.) by referencing URLs directly.
- Reducing server storage usage and eliminating the need to manage image file storage locally.

**Proposed Solution:**

Add an option to the existing media upload functionality to accept remote image URLs that are not downloaded but used directly as references. This could be implemented by:

- Allowing users to input a remote URL during image upload.
- The CMS stores the URL as a reference, without downloading or storing the file locally.

**Benefits:**

- Simplifies deployment on platforms with upload limitations.
- Reduces local storage requirements by using remote images.
- Provides flexibility for users who prefer external storage solutions.

**Possible Concerns:**

- Ensuring remote URLs are always accessible and valid.
- Potential for slower loading times depending on the source of the remote image.
  
Thank you for considering this feature request! I believe it will enhance the flexibility of PayloadCMS, especially for deployments on modern serverless hosting solutions.

**Comment by ainsleyclark** [comment]
+1 on this. Think it would be a great feature. At the moment I'm using a collection without an upload attached to it but it's becoming cumbersome.

**Comment by ainsleyclark** [comment]
Related conversation:

https://discord.com/channels/967097582721572934/967097582721572937/1292936899283128461

I think the main thing is how we send the original URL to the server. Currently there is no path that is sent via the upload request when a user pastes an image from a URL.

**Comment by Ali-9-25** [comment]
Any updates on this feature?
